data/brands/shop/supermarket.json

```diff
@@ -3082,12 +3082,14 @@
     },
     {
       "displayName": "IDEA",
```

Comment:
I notice Q114669650 is IDEA's subsdiare "IDEA Super", could you check this?

Reply:
Forgot to change that I'm sorry